public-api/src/main/java/org/pmiops/workbench/service/CriteriaService.java

```diff
@@ -0,0 +1,68 @@
+package org.pmiops.workbench.service;
+
+import org.springframework.stereotype.Service;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.pmiops.workbench.model.Criteria;
+import org.pmiops.workbench.cdr.model.DbCriteria;
+import org.pmiops.workbench.cdr.dao.CBCriteriaDao;
+import java.util.Optional;
+import org.pmiops.workbench.cdr.CriteriaMapper;
+import org.pmiops.workbench.model.CriteriaParentResponse;
+import org.pmiops.workbench.exceptions.DataNotFoundException;
+import com.google.common.collect.Multimaps;
+import com.google.common.collect.Multimap;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+public class CriteriaService {
+    private final CBCriteriaDao criteriaDao;
+    private final CriteriaMapper criteriaMapper;
+
+    @Autowired
+    public CriteriaService(CBCriteriaDao criteriaDao, CriteriaMapper criteriaMapper) {
+        this.criteriaDao = criteriaDao;
+        this.criteriaMapper = criteriaMapper;
+    }
+
+    public CriteriaParentResponse getRolledUpCounts(String conceptId, String domainId) {
```

Comment:
This method can be cleaned up and more efficient, but we'll save that for another PR>